src/libmongoc/doc/howto/source-install.rst

```diff
@@ -0,0 +1,225 @@
+    environment variable. (**Do not** use the :batch:`%AppData%` environment
+    variable.)
+
+    Despite this directory existing, it has no prescribed structure that suites
+    our purposes. As with macOS, the choice of user-local installation prefix is
+    arbitrary. This guide *strongly disrecommends* creating additional files and
+    directories directly within the user's home directory.
+
+    Consider using :batch:`%LOCALAPPDATA%\\MongoDB` as a prefix for the purposes
+    of manually installed components.
+
+
+.. _source-install.system-prefix:
+
+Selecting a System-Wide Installation Prefix
+===========================================
+
+If you wish to install the |mongo-c-driver| libraries in a directory that is
+visible to all users, there are a few very standard options.
+
+.. tab-set::
+
+  .. tab-item:: Linux, macOS, BSD, or Other Unix
+
+    Using an install `$PREFIX` of ``/usr/local/`` is the primary recommendation
+    for all Unix platforms, but this may vary on some obscure systems.
+
+    .. important::
+
+      **DO NOT** use ``/usr/`` nor ``/`` (the root directory) as a prefix: These
+      directories are designed to be carefully managed by the system. The
+      ``/usr/local`` directory is intentionally reserved for the purpose of
+      unmanaged software installation.
+
+    Alternatively, consider installing to a distinct directory that can be
+    easily removed or relocated, such as ``/opt/mongo-c-driver/``. This will be
+    easily identifiable and not interact with other software on the system
+    without explicitly opting-in.
+
+  .. tab-item:: Windows
+
+    It is **discrecommended** to manually install software system-wide on
+    Windows. Prefer instead to
+    :ref:`use a per-user unprivileged installation prefix <per-user-prefix>`.
+
+    If you wish to perform a system-wide installation on Windows, prefer to use
+    a named subdirectory of :batch:`%ProgramData%`, which does not require
+    administrative privileges to read and write. (e.g.
+    :batch:`%ProgramData%\\mongo-c-driver`)
+
+
+Installing with CMake
+*********************
+
+After you have successfully configured and built the libraries and have selected
+a suitable `$PREFIX`, you can install the built results. Let the name `$BUILD`
+refer to the directory where you executed the build (this is the directory that
+contains ``CMakeCache.txt``, amongst many other files).
+
+From a command line, the installation into your chosen `$PREFIX` can be run via
+CMake using the
+:option:`cmake --install subcommand <cmake.--install>`::
+
+  $ cmake --install "$BUILD" --prefix "$PREFIX"
+
+.. important::
+
+  If you configured the libraries while using a *multi-config generator* (e.g
+  Visual Studio, Xcode), then you will also need to pass the
+  :option:`--config <cmake--install.--config>` command-line option, and must
+  pass the same value for the build configuration which you wish to install.
```

Comment:
```suggestion
  pass the same value for the build configuration that you wish to install.
```
"That" is used to indicate a specific object, item, person, condition, etc., while "which" is used to add information to objects, items, people, situations